kolibri/core/auth/models.py

```diff
@@ -246,6 +246,14 @@ def reset_to_default_settings(self, preset=None):
             setattr(self, key, value)
         self.save()
 
+    def validate_pin_code(self):
```

Comment:
I think your changes to the JSON schema above are sufficient here - we can allow the pin_code to be null, as that what happens when the pin_code value is unset.

Reply:
Well I thought so myself, I just added that test as @bjester told me to write a regression unit test which sets the pin_code to None/null, and then calls .clean_fields() on the model

Reply:
And regarding that validate_pin_code function, should i remove it?

Reply:
@photon0205 Your test `test_null_pin_code` should assert that it does NOT raise a validation error when the pin code is null/None. Before changing the schema, that test should fail without any of these other changes, including to the `clean_fields` method or adding this method. Then once you update the schema like you have, which looks correct, that test should then start passing, thereby preventing a future regression in allowing the pin to be null/None.

Reply:
@bjester you can check the latest commit